README.md

```diff
@@ -72,9 +73,9 @@ However, a few are not fully supported:
 
 | tag                             | SDK's which implement        |
 | ------------------------------- | ---------------------------- |
-| @application.evaldelta          | Java only                    |
```

Comment:
How about the remove of `@application.evaldelta` - not running since February. @winder @jasonpaulos @ahangsu

Reply:
Is this different than here? https://github.com/algorand/algorand-sdk-testing/pull/220#discussion_r939797966

I think it can be removed.

Reply:
Nahhh.. Sorry, I was reading your answer without re-reading my question. Missing caffeine. Thx!